addons/main/functions/fnc_drawDownedUnitIndicator.sqf

```diff
@@ -8,10 +8,17 @@ private _color = [
     profileNamespace getvariable ['igui_error_RGB_B', 0.1],
     profileNamespace getvariable ['igui_error_RGB_A', 1]
 ];
+private _isMedic = (call CBA_fnc_currentUnit) getUnitTrait "Medic";
 
 {
     private _distance = _x distance _camPos;
     if (_distance > GVAR(showDownedUnitIndicatorRange)) then {continue};
     private _sizeFinal = linearConversion [0, GVAR(showDownedUnitIndicatorRange), _distance, _size, _size/10, false];
     drawIcon3D [["\a3\ui_f\data\igui\cfg\actions\bandage_ca.paa", "\A3\ui_f\data\Map\VehicleIcons\pictureHeal_ca.paa"] select (_x getVariable [QGVAR(beingRevived), false]), _color, (ASLtoAGL visiblePositionASL _x) vectorAdd [0, 0, 0.5], _sizeFinal, _sizeFinal, 0, "", 0, 0, "RobotoCondensed", "center", true, 0, 0.5];
+    if (_distance > GVAR(bleedoutTimerRange)) then {continue};
+    private _timeRemaining = round ((_x getVariable [QGVAR(bleedoutKillTime), -1]) - cba_missionTime);
+    if (_timeRemaining >= 0 && { GVAR(visibleBleedoutTimer) > 0 && { _isMedic || GVAR(visibleBleedoutTimer) == 2 } }) then {
+        private _timeString = format ["%1", _timeRemaining];
```

Comment:
Assigning variables is rather slow in arma, if it is not used multiple times then just cram in into the `drawIcon3D` directly.

you can also just use `str _timeRemaining` to convert anything to a string.

Reply:
`str _timeRemaining` wasn't working for me (just returned a blank string), but I'll try it out.

Reply:
Seems to be working.